test/unit/deref.test.js

```diff
@@ -116,7 +116,7 @@ describe('DEREF FUNCTION TESTS ', function() {
         type: 'object',
         description: 'Schema 2',
         properties: {
-          id: { default: '<long>', type: 'integer' },
+          id: { default: '<long>', type: 'integer', format: 'int64' },
```

Comment:
@webholik These format properties that are not supported were deleted as it was causing issues in AJV. Why are we not removing them anymore?